server/shared/src/test/scala/org/http4s/server/middleware/HttpsRedirectSuite.scala

```diff
@@ -23,7 +23,6 @@ import cats.syntax.all._
 import org.http4s.Uri.Authority
 import org.http4s.Uri.RegName
 import org.http4s.Uri.Scheme
-import org.http4s._
```

Comment:
This is the import I had to remove to appease tests, and I'm not sure what change has caused it to fail here (but be okay in series/0.23)